plugins/lookup/passwordstore.py

```diff
@@ -21,8 +21,14 @@
         description: query key.
         required: True
       passwordstore:
-        description: location of the password store.
-        default: '~/.password-store'
+        description:
+          - Location of the password store.
+          - 'The value is decided by checking the following in order:'
+          - If set, this value is used.
+          - If C(directory) is set, that value will be used.
+          - If the I(backend) is C(pass), C(~/.password-store) is used.
+          - If the I(backend) is C(gopass), the C(path) field in C(~/.config/gopass/config.yml) is used,
```

Comment:
Slightly more concise as:
```suggestion
          - If I(backend=pass), then C(~/.password-store) is used.
          - If I(backend=gopass), then the C(path) field in C(~/.config/gopass/config.yml) is used,
```